vaultkeeper/vaultkeeper.py

```diff
@@ -86,8 +85,14 @@ def unwrap_token(self, wrapped_token):
         return self.vault_client.token
 
     def write_credentials(self):
+        if not os.path.exists(os.path.dirname(self.configs.credential_path)):
```

Comment:
Do we want to create these dirs? I think it's reasonable to expect that the place we're writing the config to already exists.